dice/ice_comp_nuopc.F90

```diff
@@ -545,15 +569,22 @@ subroutine dice_comp_run(importstate, exportstate, target_ymd, target_tod, cosar
        call dice_datamode_ssmi_advance(exportState, importState, cosarg, flds_i2o_per_cat, &
             flux_swpf, flux_Qmin, flux_Qacc, flux_Qacc0, dt, logunit, restart_read, rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    case ('cplhist')
+       call dice_datamode_cplhist_advance(rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return 
     end select
 
     ! Write restarts if needed
+    ! TODO - no rc returned
     if (restart_write) then
        select case (trim(datamode))
        case('ssmi', 'ssmi_iaf')
           call dice_datamode_ssmi_restart_write(case_name, inst_suffix, target_ymd, target_tod, &
                logunit, my_task, sdat)
           if (ChkErr(rc,__LINE__,u_FILE_u)) return
```

Comment:
this should be an easy fix to add the rc (or remove the test for rc at line 584 and 587)